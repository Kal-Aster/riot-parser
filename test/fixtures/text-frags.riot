<text-frags>
  <p>{expr}hello</p>
</text-frags>