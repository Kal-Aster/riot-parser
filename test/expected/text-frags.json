{
  "template": {
    "type": 1,
    "name": "text-frags",
    "start": 0,
    "end": 47,
    "isCustom": true,
    "nodes": [
      {
        "type": 1,
        "name": "p",
        "start": 15,
        "end": 33,
        "nodes": [
          {
            "type": 3,
            "text": "{expr}hello",
            "start": 18,
            "end": 29,
            "expressions": [
              {
                "text": "expr",
                "start": 18,
                "end": 24
              }
            ],
            "parts": [
              "expr",
              "hello"
            ]
          }
        ]
      }
    ]
  },
  "css": null,
  "javascript": null
}